### PR TITLE
Update vertical metrics consistency

### LINKED ISSUE #93: Vertical Metrics consistency
The fonts currently published to Google Fonts don't follow the [vertical metrics specifications](https://googlefonts.github.io/gf-guide/metrics.html), resulting in inconsistency across platforms, and appearing too wide on Windows. 

In preparation for the new fonts update, they should follow the spec. However, due to the significant difference, we can't apply the strategy of inheriting the Win metrics, in which case, an overall regression must be accepted and a new compromise between platforms should be made.

Please find the current and proposed values below, with comparative images between them.

#### Current values

| Metrics         | Bungee | Hairline | Inline | Outline | Shade | Spice |
|----------------|--------|----------|--------|---------|-------|-------|
|sTypoAscender   |   860  |   860    |   860  |   860   |   860 |   860 |
|sTypoDescender  |  -140  |  -140    |  -140  |  -140   |  -140 |  -140 |
|sTypoLinegap    |   200  |   200    |   200  |   200   |   200 |   200 ||
|hhea Ascender   |   860  |   860    |   860  |   860   |   860 |   860 |
|hhea Descender  |  -140  |  -140    |  -140  |  -140   |  -140 |  -140 |
|hhea Linegap    |   200  |   200    |   200  |   200   |   200 |   200 |
|winAscent       |  1634  |  1614    |  1634  |  1644   |  1634 |  1138 |
|winDescent      |   914  |   894    |   914  |   924   |   914 |   362 |

#### Requirements and new values proposed

- Changing the Typo and hhea values, and keeping the win metrics for each family (to avoid clipping)
- Linegaps == 0
- Use of Typo Metrics _must_ be enabled

| Metrics         | Bungee | Hairline | Inline | Outline | Shade | Spice |
| ----------------|--------|----------|--------|---------|-------|-------|
| sTypoAscender   |  1020  |  1020    |  1020  |  1020   |  1020 |  1020 |
| sTypoDescender  |  -300  |  -300    |  -300  |  -300   |  -300 |  -300 |
| sTypoLinegap    |     0  |     0    |     0  |     0   |     0 |     0 |
| hhea Ascender   |  1020  |  1020    |  1020  |  1020   |  1020 |  1020 |
| hhea Descender  |  -300  |  -300    |  -300  |  -300   |  -300 |  -300 |
| hhea Linegap    |     0  |     0    |     0  |     0   |     0 |     0 |
| winAscent       |  1634  |  1614    |  1634  |  1644   |  1634 |  1138 |
| winDescent      |   914  |   894    |   914  |   924   |   914 |   362 |

#### Comparative images
Old = GF distributed / New = proposed values

**Windows**
Win-before-after

**Mac**
Mac-before-after

### PR DESCRIPTION
**Pull request by martelahaie**
Improve vertical metrics consistency, according to the new values proposed in issue #93. This fixes #93.

### CODE REVIEW

**Review by justvanrossum — COMMENTED** [review]
Looks good to me! One nit: even though it is out of scope for #93, please also apply the computeWinAscentDescent function to the rotated and color fonts, in assembleRotatedSources.py and in assembleColorSources.py.

File: sources/1-drawing/Bungee-Regular.ufo/lib.plist [hunk 1]
```diff
@@ -4158,6 +4158,8 @@
     <integer>1</integer>
     <key>com.typemytype.robofont.compileSettings.checkOutlines</key>
     <integer>1</integer>
+    <key>com.typemytype.robofont.compileSettings.createDummyDSIG</key>
```
Comment (justvanrossum) [inline sources/1-drawing/Bungee-Regular.ufo/lib.plist:4161]: Was there a reason for this change? I don't think it influences the build.

File: scripts/assembleTools.py [hunk 2]
```diff
@@ -4,6 +4,20 @@
 from fontTools.pens.transformPen import TransformPointPen
 
 
+def computeWinAscentDescent(font):
```
Comment (justvanrossum) [inline scripts/assembleTools.py:7]: Please move this function to the end of the module.

**Review by justvanrossum — COMMENTED** [review]
I left two minor comments.

File: sources/1-drawing/Bungee-Regular.ufo/lib.plist (hunk 1 above)
Comment (martelahaie) [reply sources/1-drawing/Bungee-Regular.ufo/lib.plist:4161]: This changed automatically when setting the new metrics in Robofont. It doesn't seem to influence the build so we can omit this commit.

**Pull request merged**